parlai/mturk/tasks/acute_eval/example_script.py

```diff
@@ -0,0 +1,54 @@
+from parlai.mturk.tasks.pairwise_dialogue_eval.run import main as run_main, add_args
+
+"""
+Example script for running ACUTE-EVAL.
+The only arguments that need to be modified for this to be run are:
+'dialogs_path':  Path to folder containing logs of all model conversations.
+    Each model should have one file named '<modelname>.jsonl'
+'model_comparisons': List of tuples indicating pairs of models to be compared.
+    The model names here should match the names of files in the folder
+'onboarding_tasks': List of tuples in format (id1, id2, name of matchup) where
+    id1, id2 are conversation ids where id2 is the correct conversation's id
+Help strings for the other arguments can be found in run.py
+"""
+
+
+def set_args():
+    args = add_args()
+    args['dialogs_path'] = '/home/ParlAI/parlai/mturk/tasks/acute_eval/example'
+    args['model_comparisons'] = [('modela', 'modelb')]
+    args['onboarding_tasks'] = [('ZYX', 'AGHIJK', 'example_qual')]
+    args['task_description'] = {
+        'num_subtasks': 2,
```

Comment:
i think this should be the existing arg, `subtasks_per_hit`, instead of `num_subtasks`.

Reply:
EDIT: this is why i found that issue with the Optional feedback not submitting.